bentoml/grpc/types.py

```diff
@@ -0,0 +1,108 @@
+# pragma: no cover
+"""
+Specific types for BentoService gRPC server.
+"""
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from typing import Any
+    from typing import Type
+    from typing import Literal
+    from typing import TypeVar
+    from typing import Callable
+    from typing import Optional
+    from typing import Annotated
+    from typing import Awaitable
+    from typing import NamedTuple
```

Comment:
Why are we doing this instead of `import typing as t`?

Reply:
Oh let me change it back since this is auto-complete with pyright 😄